auth-api/src/main/java/io/stargate/auth/api/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright The Stargate Authors
```

Comment:
qq: which file(s) require this license at the top?

Reply:
Any java file that we've created